src/test/unit/cpu.test.js

```diff
@@ -57,21 +53,21 @@ describe('processThreadCPUDelta', function () {
       0.2,
     ]);
     expect(processedThread1.samples.threadCPUDelta).toEqual([
-      0.1, 0.1, 0.1, 0.2, 0.2, 0.2,
+      0.1, 0, 0, 0, 0, 0.2,
     ]);
```

Comment:
After checking 1000 profiles, I have found the following:

 - There can be more than one `null` value at the start of the array.
 - In every checked profile, once you see a non-`null` value, the rest of the values are also non-`null`.

And the reason for multiple `null` values at the beginning appears to be that we never implemented threadCPUDelta in the base profiler! So during process startup, you get all nulls until the non-base profiler takes over. Example: https://share.firefox.dev/4iG7vsj

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1756519

Reply:
Given Florian's confusion in [bug 1756519](https://bugzilla.mozilla.org/show_bug.cgi?id=1756519), I would say zeroing out the base profiler values is preferable over what we do today: It'll be obviously wrong rather than plausible but confusing.

Reply:
thanks for the investigation, this sounds good to me!

Reply:
Actually, after talking to Florian more, I'm no longer convinced that setting the base profiler values to zero is the better outcome. [Before](https://share.firefox.dev/402FS5D), [after](https://deploy-preview-5265--perf-html.netlify.app/public/a8m3v7dytdkd8vcyrhdw8jbk4bcsws9p1k8xk7g/calltree/?globalTrackOrder=81w405w7&hiddenGlobalTracks=1w46&hiddenLocalTracksByPid=10084-0wyvz1wz7~1636-0w9~9488-0wi~4600-0wi~1268-0wi~5596-0w57wx0~5476-0wx8~12148-0wx6&localTrackOrderByPid=10084-z80wz7z9wzk~1636-0w9~9488-0wi~4600-0wi~1268-0wi~5596-0wx3~5476-0wx8~12148-x70wx6x8wxe&range=m989&thread=0&v=10). I think I'll just update this PR to avoid the behavior change.

Reply:
I've removed the first commit. This has the effect that, since `getMaxThreadCPUDeltaPerMs` no longer looks at the processed CPU deltas, it might return a value that's lower than the maximum cpu delta per ms once the null substitution has happened:
Let's say you have this unprocessed data: `[{ timeDelta: 1, cpuDelta: null }, { timeDelta: 10, cpuDelta: 10 }]`
Then you replace the `null` with the 10.
Now the first value has `{ timeDelta: 1, cpuDelta: 10 }`, i.e. 10 CPU per ms. `getMaxThreadCPUDeltaPerMs` will now just compute 1 CPU per ms.
I think that's ok because this situation is broken anyway.